### DISCUSSION TITLE
Standup (Tuesday, May 24th)

### DISCUSSION THREAD

**Discussion by antonieto** [CarlosCRG19/ijalti-peers#70] [General]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:

```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
 ```

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - Fixed a redundancy issue with the help of @CarlosCRG19 
 - Started Auth middleware with @BrandonMagana 

**What are your priorities for the next 24 hours?**:
 - Help protect some endpoints 

**Blockers**:
 - None right now

**Shoutouts**:
 - @CarlosCRG19 for helping me solve the redundancy issue and @BrandonMagana for coding his first middleware

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
 - Finished endpoints for aspirants login and signup

**What are your priorities for the next 24 hours?**:
 - Save token in front
 - Meet with @CristianCazares and @DavBot02 

**Blockers**:
 - None

**Shoutouts**:
 - @CristianCazares @DavBot02

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Cleaned autocomplete code

**What are your priorities for the next 24 hours?**:
 - Understand what Carlos is talking about

**Blockers**:
 - What Carlos is talking about

**Shoutouts**:
 - @CristianCazares @CarlosCRG19

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
 - Started working on authentication middleware with @antonieto 

**What are your priorities for the next 24 hours?**:
 -  finish this middleware

**Blockers**:
 - error with typescript

**Shoutouts**:
 - @antonieto  for helping me

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
 - Meet to start the login stuff

**What are your priorities for the next 24 hours?**:
 - Save the token on frontend and understand it.

**Blockers**:
 - My optatives evidence is due today

**Shoutouts**:
 - @CarlosCRG19
